docs/DeveloperGuide.md

```diff
@@ -655,20 +689,38 @@ if there is no existing same item. If there is an existing same item,
 
 1. Adding an item
     1. Test Case: `add-i n/Chicken q/123 s/NTUC`
-       Expected: Item with `Name` of Chicken, `Quantity` of 123 and `Supplier` of NTUC added 
-       
+       Expected: Item with `Name` of Chicken, `Quantity` of 123 and `Supplier` of NTUC added
+
     1. Test Case: `add-i n/Chicken q/123 s/giant max/500 metric/kg`
-       Expected: Item with `Name` of Chicken, `Quantity` of 123, `Supplier` of NTUC, `MaxQuantity` of 500 and `Metric` of kg added when there is no existing same item. 
+       Expected: Item with `Name` of Chicken, `Quantity` of 123, `Supplier` of NTUC, `MaxQuantity` of 500 and `Metric` of kg added when there is no existing same item.
 
 ### Adding to an existing item
 
 1. Adding to an existing item
     1. Test Case: `add-i n/Chicken q/123 s/NTUC`
        Expected: Item with `Name` of Chicken and `Supplier` of NTUC will have it's `Quantity` combine with input item's `Quantity`. `MaxQuantity` `Tags` `Metric` will be adopted from the existing item.
-       
+
     1. Test Case: `add-i n/Chicken q/123 s/giant max/500 metric/kg`
        Expected: User will receive an error message as `MaxQuantity` or `Metric` should not be defined when adding to existing item.
 
+### Editing Command
+
+1. Editing an item or delivery
+
+    1. Prerequisites: List all item or delivery using `list-i` and `list-d` respectively.
+
+    1. Test Case: `edit-i 1 n/TUNA` <br>
```

Comment:
Would you like to edit an empty tag? Change metric and max quantity? Since these are the ones that are more special for edit-i

Reply:
Makes sense, will update it